homeassistant/components/camera/arlo.py

```diff
@@ -158,15 +161,48 @@ def disable_motion_detection(self):
         self._motion_status = False
         self.set_base_station_mode(ARLO_MODE_DISARMED)
 
+    @staticmethod
+    def clean_attr(attr):
+        """Return unknown if attribute is None (non-subscriptable)."""
+        return str(attr) if attr is not None else STATE_UNKNOWN
+
     def update(self):
         """Add an attribute-update task to the executor pool."""
-        self.attrs[ATTR_BATTERY_LEVEL] = self._camera.get_battery_level
-        self.attrs[ATTR_BRIGHTNESS] = self._camera.get_battery_level
-        self.attrs[ATTR_FLIPPED] = self._camera.get_flip_state,
-        self.attrs[ATTR_MIRRORED] = self._camera.get_mirror_state,
-        self.attrs[
-            ATTR_MOTION] = self._camera.get_motion_detection_sensitivity,
+        # pylint: disable=W0212
```

Comment:
Not allow to disable. Thats make uggly workarounds

Reply:
@tchellomello Should we add a method to the PyArlo library classes to retrieve the session on the objects (and properties like refresh_rate and last_refresh). That would fix these lint issues (protected access).

Reply:
@jwillaz  +1 yes, let's implement. Do you want to submit a PR?

Reply:
Yes the correct work flow is: update module library -> update home-assistant

Reply:
@jwillaz  PR https://github.com/tchellomello/python-arlo/pull/60 created on `pyarlo` library.

Reply:
@jwillaz  https://pypi.python.org/pypi/pyarlo/0.0.9 has been published.

Reply:
Implemented by https://github.com/tchellomello/python-arlo/pull/61